hexa/pipelines/graphql/schema.graphql

```diff
@@ -188,7 +188,7 @@ extend type Mutation {
   runPipeline(input: RunPipelineInput): RunPipelineResult! @loginRequired
   pipelineToken(input: PipelineTokenInput): PipelineTokenResult! @loginRequired
   uploadPipeline(input: UploadPipelineInput): UploadPipelineResult! @loginRequired
-  logPipelineMessage(input: LogPipelineMessageInput): LogPipelineMessageResult!
-  updatePipelineProgress(input: UpdatePipelineProgressInput): UpdatePipelineProgressResult!
-  addPipelineOutput(input: AddPipelineOutputInput): AddPipelineOutputResult!
+  logPipelineMessage(input: LogPipelineMessageInput): LogPipelineMessageResult! @loginRequired
```

Comment:
I'd rather not add the loginRequired on those for now. I'll check with Jeremie what we'll do.

Reply:
Okay, I'll revert it